README.md

```diff
@@ -99,6 +99,14 @@ specs. This piece of code does the job:
 (ad-activate 'rspec-compile)
 ```
 
+### Vagrant
+
+You can run specs inside a vagrant box. You can enable it through the
```

Comment:
I think Vagrant should always be capitalized when it's not the name of the executable.